bin/select_mock_targets

```diff
@@ -69,19 +69,19 @@ if args.healpixels is None:
     healpixels = desimodel.footprint.tiles2pix(args.nside, tiles)
 else:
     healpixels = np.array(args.healpixels)
-
+    
 if args.overwrite:
     log.info('Processing {} pixel(s).'.format(len(healpixels)))
 else:
     keep = list()
     for i, pixnum in enumerate(healpixels):
-        truthfile = mockio.findfile('truth', args.nside, pixnum, basedir=args.output_dir)
+        truthfile = mockio.findfile('truth', args.nside, pixnum, basedir=args.output_dir, obscon='dark')
         if not os.path.exists(truthfile):
             keep.append(i)
 
-    log.info('{}/{} pixels remaining to do'.format(len(keep), len(healpixels)))
+    log.info('{}/{} pixels remaining to do. {}'.format(len(keep), len(healpixels), healpixels[keep]))
     healpixels = healpixels[keep]
```

Comment:
I don't understand the intent of the third number.

Reply:
This was a lazy way for me know the healpixels to actually be processed, as opposed to the ones I requested, given some had already been generated and were on disk.  I was playing games like submitting the second half of that list to a separate job to run in tandem.  Obviously, there are more sophisticated approaches and it doesn't need to be kept.